tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -421,7 +430,36 @@ install_cc_tdx_kernel() {
 
 install_cc_sev_kernel() {
 	kernel_version="$(yq r $versions_yaml assets.kernel.sev.version)"
-	install_cc_tee_kernel "sev" "${kernel_version}"
+	
+	export kernel_version=${kernel_version}
+	export KATA_BUILD_CC=yes
+	config_version=$(get_config_version)
+	module_dir="${builddir}/kata-linux-${kernel_version#v}-${config_version}/lib/modules/${kernel_version#v}"
+	
+	install_cached_component \
+		"kernel" \
+		"${jenkins_url}/job/kata-containers-2.0-kernel-sev-cc-$(uname -m)/${cached_artifacts_path}" \
+		"${kernel_version}" \
+		"$(get_kernel_image_name)" \
```

Comment:
Do we want the kernel image name or the last commit of the image?

Reply:
I am following what TDX is doing for the kernel here: https://github.com/kata-containers/kata-containers/blob/ad889489a095d73e8d1bdb813533f72b43e6357e/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L405
And I also believe get_kernel_image_name return the last modified commit of the image from here: https://github.com/kata-containers/kata-containers/blob/ad889489a095d73e8d1bdb813533f72b43e6357e/tools/packaging/scripts/lib.sh#L197

Reply:
Yup, you're correct! :-)